jbpm-task-assigning/jbpm-task-assigning-core/src/main/java/org/jbpm/task/assigning/model/solver/TaskDifficultyComparator.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.task.assigning.model.solver;
+
+import java.io.Serializable;
+import java.util.Comparator;
+
+import org.apache.commons.lang3.builder.CompareToBuilder;
+import org.jbpm.task.assigning.model.Task;
+
+public class TaskDifficultyComparator implements Comparator<Task>,
+                                                 Serializable {
+
+    @Override
+    public int compare(Task a, Task b) {
```

Comment:
One other change we've also done over the last several weeks in the `optaplanner-examples` codebase was to convert all `Comparator`s from `CompareToBuilder` to static `Comparator.comparing(...)`.

It performs better, which will be important in the context of a server app, and also doesn't require a dependency on an external library.